gallery/src/components/NxTabs/NxTabsTileHeaderExample.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel
+} from '@sonatype/react-shared-components';
+
+const NxTabsSimpleExample = () => {
+  const [activeTabId, setActiveTabId] = useState<number>(0);
+
+  function handleTabSelect(index: number) {
+    setActiveTabId(index);
+  }
+
+  return (
+    <div className="nx-tile">
+      <div className="nx-tile-header">
+        <div className="nx-tile-header__title"><h2 className="nx-h2">Tile Header</h2></div>
+      </div>
```

Comment:
Going forward I'm trying to use the semantic elements in all tile examples.  We should get around to updating the old examples at some point too.
```suggestion
    <section className="nx-tile">
      <header className="nx-tile-header">
        <div className="nx-tile-header__title"><h2 className="nx-h2">Tile Header</h2></div>
      </header>
```